src/libopenrave/kinbody.cpp

```diff
@@ -65,6 +65,21 @@ inline void _ResizeVectorFor2DTable(std::vector<int8_t>& vec, size_t vectorSize)
     }
 }
 
+static void _PrintDOFValuesForInitialLinkTransformations(const KinBody& body, const std::vector<dReal>& vdoflastsetvalues, const char* context)
+{
+    if( body.GetDOF() == 0 ) {
+        return;
+    }
+    std::stringstream ssJoints;
+    for(size_t iDOF = 0; iDOF < vdoflastsetvalues.size(); ++iDOF) {
+        if( iDOF > 0 ) {
+            ssJoints << ",";
+        }
+        ssJoints << vdoflastsetvalues[iDOF];
+    }
+    RAVELOG_INFO_FORMAT("env='%s', body '%s' _vInitialLinkTransformations is updated in %s by dofValues=[%s]", body.GetEnv()->GetNameId()%body.GetName()%context%ssJoints.str());
```

Comment:
Very minor, but let's print `env=%s` instead of `env='%s'` (no single quotes) so that log messages are consistent with other places.

Reply:
oops, right!

since many of printing like `KinBody`'s `GetName` has single quotes, I misunderstood that we need quotes for env.

thanks~